pkg/kn/commands/common/human_readable_flags.go

```diff
@@ -95,6 +83,54 @@ func ServiceGetHandlers(h hprinters.PrintHandler) {
 	h.TableHandler(kServiceColumnDefinitions, printKServiceList)
 }
 
+// RevisionGetHandlers adds print handlers for revision get command
```

Comment:
This should live with the revision package.

Reply:
Let me address this tomorrow. Assuming you are still OK with last version. That passes all tests. See my last comment for details on what I had to do to get tests passing...

Reply:
I agree that revision and versions specific code should go into their respective packages. Maybe we can get rid of `commons` alltogether ? `types.go` could potentially moved to `root.go` again (see https://github.com/knative/client/pull/145#issuecomment-498192891 how I think this could work), and the other code could be moved to d "commands" package, too (like for the root command and the content of "etc")